### ISSUE TITLE
Overlays Slowmo option not working when pressed.

### ISSUE THREAD

**Issue by blackman91** [libretro/RetroArch#6785]
## Description
The slowmo option does not work in the retropad overlay: 
![screenshot_20180516-065301](https://user-images.githubusercontent.com/6578180/40115973-c99303c6-58d7-11e8-9b35-783964b58fdd.png)

They do not work on any overlay that has those options too. The input hotkey for slowmo is working with a gamepad but that can't be mapped to Overlays. The other options shown in the screenshot do work when pressed.

### Expected behavior
Pressing those buttons do the action.

### Actual behavior
When pressed they don't do anything.

### Steps to reproduce the bug

1. Press the options wheel in the retropad overlay.
2. Press slowmo or rewind.

### Bisect Results

RetroArch 1.7.3

### Version/Commit
You can find this information under Information/System Information

- RetroArch: 1.7.3

### Environment information

- OS: Android 7.

**Comment by fr500** [comment]
do you have achievements enabled?

**Comment by blackman91** [comment]
Nope, no achievements enabled.

**Comment by robotjonboy** [comment]
Same issue for me. Retroarch 1.7.3. Android 6. I was able to get the rewind overlay button to work by enabling rewind on the rewind menu, but the slow motion overlay button still does not work.

**Comment by blackman91** [comment]
Maybe it is related to this issue:
https://github.com/libretro/RetroArch/issues/6773

I double checked a no it still doesn't work on the overlay even with achievements disabled.

**Comment by robotjonboy** [comment]
I have achievements disabled as well. Still no luck.

**Comment by blackman91** [comment]
Yeah if you enable rewind the overlay rewind button works so it is only the slow mo overlay button that is not working at all even on latest nightly.

**Comment by zoltanvb** [comment]
Checked, slow motion button works nowadays.
